generative_data_prep/utils/utils.py

```diff
@@ -46,6 +41,13 @@ def data_prep_arg_builder(parser: argparse.ArgumentParser):
             " running tokenization"
         ),
     )
+    parser.add_argument(
+        "--log_file_path",
+        type=str,
+        required=False,
+        default=None,
```

Comment:
should the default specified in `help` go here?

Reply:
I don't know how to do that, since we don't have access to the output directory here, I set that default later on in the code

Reply:
help="The log file path. Defaults to '<output_path>/logs.log'",